src/hoprd_ingress.rs

```diff
@@ -84,3 +87,147 @@ pub async fn delete_ingress(client: Client, name: &str, namespace: &str) -> Resu
         Ok(println!("[INFO] Ingress {name} in namespace {namespace} about to delete not found"))
     }
 }
+
+
+/// Creates a new Ingress for accessing the hoprd node,
+///
+pub async fn open_port(client: Client, service_namespace: &str, service_name: &str, ingress_config: &IngressConfig) -> Result<String, HoprError> {
+    let namespace = ingress_config.namespace.as_ref().unwrap();
+    let api: Api<ConfigMap> = Api::namespaced(client.clone(), namespace);
+    let port: String = get_port(client.clone(), ingress_config).await.unwrap();
+    let pp = PatchParams::default();
+    let patch = json!({
+           "data": {
+                port.to_owned() : format!("{}/{}:{}", service_namespace, service_name, port.to_owned())
+            }
+        });
+    match api.patch("ingress-nginx-tcp", &pp, &Patch::Merge(patch.clone())).await {
+            Ok(_) => {},
+            Err(error) => {
+                println!("[ERROR]: {:?}", error);
```

Comment:
Please, use either the `log` crate, or better yet, the `tracing`  crate for information output.

Reply:
Added the tracing crate